bridge_adaptivity/prod_run.sh

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
-
-python manage.py migrate --settings=config.settings.prod
-python manage.py collectstatic -v 3 -c --noinput --settings=config.settings.prod
```

Comment:
Could you please point me on which step do we collect static within current changes?

Reply:
Static is collected when we build new images. You can see this step in the Dockerfile: https://github.com/harvard-vpal/bridge-adaptivity/pull/233/files#diff-6da744b8e40a699166501279107d566cR37

Reply:
I added collecting static in the local machine when it's starting.
it's script:
https://github.com/harvard-vpal/bridge-adaptivity/pull/233/files#diff-6e3f0ef6ecb83cb1a9a0931ed82e5c4bR2
in this line it is used: 
https://github.com/harvard-vpal/bridge-adaptivity/pull/233/files#diff-6da744b8e40a699166501279107d566cR46